crates/wash-lib/src/parser/mod.rs

```diff
@@ -542,14 +547,17 @@ struct RawDevConfig {
     /// Top level override of the WADM application manifest(s) to use for development
     ///
     /// If unspecified, it is up to tools to generate a manifest from available information.
-    pub manifests: Option<Vec<DevManifestComponentTarget>>,
+    pub manifests: Option<OneOrMore<DevManifestComponentTarget>>,
 
     /// Configuration values to be set up
     #[serde(alias = "configs")]
-    pub config: Option<Vec<DevConfigSpec>>,
+    pub config: Option<OneOrMore<DevConfigSpec>>,
 
     /// Configuration values to be set up
-    pub secrets: Option<Vec<DevSecretSpec>>,
+    pub secrets: Option<OneOrMore<DevSecretSpec>>,
```

Comment:
What is `OneOrMore` coming from here? I think I can understand that this means that there should be at least one. But instead of an Option<OneOrMore>, maybe a Vec that is empty?

Reply:
Ahh see https://github.com/wasmCloud/wasmCloud/pull/3264#discussion_r1801583052 -- this is to enable `serde` (and resultingly TOML deserialization), so we can have config similar to the original feature proposal